### PR TITLE
Added cron config for bulk list create for demo 01 cluster

### PR DESCRIPTION
**Pull request by nitinprabhu**

### CODE REVIEW

**Review by sunilnatikar — APPROVED** [review]

File: k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (timja) [inline k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml:1]: this needs to be referenced somewhere

Comment (nitinprabhu) [reply k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml:1]: I am not sure where we need to reference  `k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml` could you please point me to the config ?

Comment (timja) [reply k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml:1]: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/cluster-01-overlay/kustomization.yaml

Comment (nitinprabhu) [reply k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml:1]: thanks @timja

**Review by timja — APPROVED** [review]

**Pull request merged**